### DISCUSSION TITLE
Use local json files in the backend

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [wailsapp/wails#2402] [Q&A]
I am using some json files in the backend (files that I do not fetch from an api somewhere else, they are already present in the project). Is there a way to use them ? 

I can use them normally in dev mode (by giving the path to the deserializer), but in release mode, they are not available anymore (as the containing folder obviously doesn't exist anymore)

**Comment by leaanthony** [comment]
Go's embed is for this.

**Reply by FrenchGithubUser** [reply]
Indeed, thanks !

**Comment by FrenchGithubUser** [comment]
I figured out that this is not necessarily the solution I'm looking for. Is there an API to store some files on the machine ? (like a folder in `.config` on linux ?

**Reply by leaanthony** [reply]
Why not just write the files to disk from an embed if they don't exist?

**Reply by FrenchGithubUser** [reply]
It seemed to be more complicated than having the files in a folder like .config. But I'll give it a try !

**Reply by leaanthony** [reply]
Let us know if you find an easy way to package Linux binaries

**Reply by FrenchGithubUser** [reply]
Some of the files I store also need to be updated (user configuration). Is there a way to update them even if they are embedded or should I copy the base configuration and edit the copy ? (in this case, where should I store the configuration, in .config ?)

Edit : I think it is better to copy the base config somewhere and then edit the copy (otherwise updates would erase it). But then, is there a specific place I should store the config (for each OS) ?

Edit2 : Looks like os.UserConfigDir() is the place
